PayBitForward/Messaging/Messages/Acknowledge.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+
+namespace PayBitForward.Messaging
+{
+    public class Acknowledge : Message
+    {
+        /// <summary>
+        /// Status information
+        /// </summary>
+        public string StatusInformation { get; set; }
+
+        public Acknowledge(MessageType messageId, Guid convoId, string statusInformation) : base(messageId, convoId)
```

Comment:
In the constructor we can just set the MessageType to MessageType.Ackknowledge or something like that so it doesn't need to be in the constructor.